bitmap_commands.go

```diff
@@ -127,3 +129,21 @@ func (c cmdable) BitField(ctx context.Context, key string, values ...interface{}
 	_ = c(ctx, cmd)
 	return cmd
 }
+
+// BitFieldRO - Read-only variant of the BITFIELD command.
+// It is like the original BITFIELD but only accepts GET subcommand and can safely be used in read-only replicas.
+// - BitFieldRO(ctx, key, "<Encoding0>", "<Offset0>", "<Encoding1>","<Offset1>")
+func (c cmdable) BitFieldRO(ctx context.Context, key string, values ...interface{}) *IntSliceCmd {
+	args := make([]interface{}, 2, 2+len(values))
+	args[0] = "BITFIELD_RO"
+	args[1] = key
+	if len(values)%2 != 0 {
+		panic("BitFieldRO: invalid number of arguments, must be even")
```

Comment:
This doesn't feel like the rest of the library. Maybe we should do nothing here? WDYT?

Reply:
I just followed https://github.com/redis/go-redis/blob/3c56d748ef83ac14cd487e4f3f7b91214c6f3ab6/bitmap_commands.go#L100C30-L100C30

Reply:
Gah.. I'd wager the whole thing can change; not in this PR (fair). I don't love all these panics.

Reply:
Yeah, me too. I'll open an issue for this